src/app/pages/ArticlePage/index.jsx

```diff
@@ -83,10 +117,11 @@ const ArticlePage = ({ pageData, mostReadEndpointOverride }) => {
           />
         </GhostGrid>
       </StyledMain>
-      <MostReadContainer
-        mostReadEndpointOverride={mostReadEndpointOverride}
-        constrainMaxWidth
-      />
+      <ConstrainedMostReadSection>
```

Comment:
Thanks for putting the styles in the page type where they are used. 
Wondering if we should now rename this wrapper to `ArtcilePageMostReadSection` to match with the `FrontPageMostReadSection`?

Reply:
> Opted to revert the changes to put the styles in the page type where they are used, because it was starting to look like a major refactor as the all of the MostReadContainer related testing would have had to be updated as well.

Due to the revert this is no longer possible